scripts/getScramArch_EnvVar.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
At the moment this is the script used instead of getScramArch.sh.
https://github.com/ebecheva/HGCTPGValidation/blob/93da251d0c313ad3b1c2922e3be8f0e900256fd2/Jenkinsfile#L98

Reply:
Then only one of the two scripts should be kept if the other is not used.
And the file name ` getScramArch.sh` should be preferred since it is not clear what `_EnvVar` in the name means.

Reply:
This script has been removed. In Jenkinsfile it has been replaced by getScramArch.sh.